### PR TITLE
Final changes made to Portfolio website: Mobile

### PR DESCRIPTION
**Pull request by evansnyamekye**
Portfolio mobile website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,270 +1,222 @@
...
-                src="images/arrows/Separator Bottom.png"
-                alt="horizontal arrow separater"
-              />
-            </li>
-          </ul>
-        </div>
-      </div> 
+    </div>
 
       <div class="contact_container">
```
Comment (hiromataba) [inline index.html:221]: - [ ] Please add required to the text-area as required for validation purpose

image

Comment (hiromataba) [inline index.html:221]: - [ ] Please add a POST method to the form and also use [ formspree service ](https://formspree.io/html) 
image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @evansnyamekye and @evansnyamekye     :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title   :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTOINAL] Please update the placeholder of your contact form 😉 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @evansnyamekye, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**